src/megaclient.cpp

```diff
@@ -1525,7 +1530,7 @@ void MegaClient::exec()
                             if (fa->th == UNDEF)
                             {
                                 // client app requested the upload without a node yet, and it will use the fa handle
-                                app->putfa_result(fah, fa->type, nullptr);
+                                app->putfa_result(fah, fa->type, API_OK);
```

Comment:
Good one! I wonder how that `nullptr` got there, instead of an `error` :)

Reply:
There used to be two different versions of putfa_result, one for errors and one for success.  But, we never did anything with the data passed by pointer, just called the client app back with API_OK.